roles/lib_openshift/src/test/unit/test_oc_project.py

```diff
@@ -37,11 +37,9 @@ class OCProjectTest(unittest.TestCase):
         'description': 'All things operations project',
     }
 
-    @mock.patch('oc_project.locate_oc_binary')
-    @mock.patch('oc_project.Utils.create_tmpfile_copy')
-    @mock.patch('oc_project.Utils._write')
+    @mock.patch('oc_project.Utils', autospec=True)
```

Comment:
@kwoodson, speccing and autospeccing is still a good thing, we can look into adding it to new code during code reviews.